department-of-veterans-affairs/caseflow
# Queue | Fix sending ama appeal issues in AttorneyCaseReview payload

### Description
In #7007 we updated the attorney checkout flow to support AMA cases. However, I made an incorrect assumption when building the AttorneyCaseReview payload in SubmitDecisionView (that `SubmitDecisionView` will only be used for legacy cases), resulting in always sending `issues: []` to the server.

### Acceptance Criteria
- [ ] AMA Issue dispositions / remand reasons are set correctly during Attorney checkout flow

### Testing Plan
1. Go to `/queue` as `BVASCASPER1`
2. Complete Draft Decision checkout flow on AMA appeal
3. Confirm `issues` in payload / server
4. For manual confirmation, switch to judge assignee user, open case details, confirm issue dispositions
